package.json

```diff
@@ -15,8 +15,8 @@
     "@chakra-ui/react": "^2.4.6",
     "@emotion/react": "^11",
     "@emotion/styled": "^11",
-    "@railgun-community/quickstart": "3.9.2",
-    "@railgun-community/shared-models": "3.7.5",
+    "@railgun-community/quickstart": "3.8.12",
```

Comment:
Odd that we had to downgrade, wonder how they are managing the versioning

Reply:
Yes, I think they messed up their versioning, there are a couple 3.9 and one 3.10, but this is the latest